lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -611,6 +611,12 @@ SPIRVFunction *LLVMToSPIRVBase::transFunctionDecl(Function *F) {
   if (BM->isAllowedToUseExtension(ExtensionID::SPV_INTEL_fpga_buffer_location))
     BufferLocation = ((*F).getMetadata("kernel_arg_buffer_location"));
 
+  // Translate runtime_aligned metadata if it's attached to the translated
+  // function declaration
+  MDNode *RuntimeAligned = nullptr;
+  if (BM->isAllowedToUseExtension(ExtensionID::SPV_INTEL_runtime_aligned))
+    RuntimeAligned = ((*F).getMetadata("kernel_arg_runtime_aligned"));
```

Comment:
Wouldn't this still work? Or is some type magic necessary here?
```suggestion
    RuntimeAligned = F->getMetadata("kernel_arg_runtime_aligned");
```